Some gcc versions do not provide this file anymore and this copy was a
workaround for other versions. So let's condition the copy to the existence
of the file.

--- ./scripts/build/libc/glibc-eglibc.sh-common.orig	2013-10-03 11:07:29.903994675 +0200
+++ ./scripts/build/libc/glibc-eglibc.sh-common	2013-10-03 11:07:57.631994599 +0200
@@ -404,7 +404,7 @@
             # but it breaks arm, see http://sourceware.org/ml/crossgcc/2006-01/msg00091.html
             case "${CT_ARCH}" in
                 arm)    ;;
-                *)  CT_DoExecLog ALL cp -v "misc/syscall-list.h"            \
+                *)  [ -e misc/syscall-list.h ] && CT_DoExecLog ALL cp -v "misc/syscall-list.h"            \
                                            "${CT_HEADERS_DIR}/bits/syscall.h"
                     ;;
             esac
